### PR TITLE
Automated bugfix APPTRACK-1332 Vacancy not showing for assigned Exec Secretary in Your Vacancies tab

### PR DESCRIPTION
**Pull request by lfcabrera14**

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature [hunk 1]
```diff
@@ -499,8 +496,14 @@ Would you like to log in now or go back to the home page?
...
       | SSJTest   | Michelle   | Pololi   | mario@gmail.com | 2018212343 | 2023323454    | Masters          | 7 Mills | 12378     | Reston | VA    | USA     | 20453 | Fred                  | Trout                  | Laste                | referenceOne@test.com | 7036875816              | Supervisor/Manager       | PROJECT MANAGER           | TESTING ORGANIZATION FIELD FOR REFERENCE ONE | John                  | Steven                 | Vakhutko             | referenceTwo@test.com | 7036875816              | TEST AUTOMATION LEAD      | TESTING ORGANIZATION FIELD FOR REFERENCE TWO | Elizabeth                    | Mary                          | Smith                       | updatedReferenceOne@test.com | 7036875817                     | Colleague                       | BUSINESS ANALYST                 | UPDATED TESTING ORGANIZATION FIELD FOR REFERENCE ONE | Adam                         | Elliot                        | Washington                  | updatedReferenceTwo@test.com | 7036875817                     | PRODUCT OWNER                    | UPDATED TESTING ORGANIZATION FIELD FOR REFERENCE TWO |
 
+  @TC_APPTRACK-827 @TODO
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:499]: I would tag it as InProgress than TODO and add userid

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java [hunk 2]
```diff
@@ -3,10 +3,16 @@
...
 import ServiceNow.PlatformBusinessApps.SSJ.playwright.StepsImplementation.ApplicantProfileStepsImpl;
 import ServiceNow.PlatformBusinessApps.SSJ.playwright.StepsImplementation.Reset_Account_StepsImpl;
+import appsCommon.PlaywrightUtils.Playwright_Common_Utils;
 import appsCommon.PlaywrightUtils.Playwright_ServiceNow_Common_Methods;
+import com.microsoft.playwright.ElementHandle;
+import com.nci.automation.utils.CucumberLogUtils;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:13]: Please remove the empty line

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
+import appsCommon.PlaywrightUtils.Playwright_Common_Utils;
+import com.microsoft.playwright.ElementHandle;
 import com.microsoft.playwright.Page;
 import com.microsoft.playwright.options.AriaRole;
+import com.nci.automation.utils.CucumberLogUtils;
+import com.nci.automation.utils.MiscUtils;
+import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java:17]: Please remove the empty line

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java:21]: Please remove the empty line

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java [hunk 5]
```diff
@@ -665,4 +687,37 @@ public static void user_adds_duplicate_committee_member_for_executive_secretary(
         page.locator(Vacancy_Committee_Page.vacancyCommitteeMemberRoleDropDown ).click();
         page.waitForSelector(Playwright_Common_Locators.dynamicTextLocator("Executive Secretary (non-voting)")).click();
     }
+
+    public static void impersonate_holly_or_any_vacancy_manager(){
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java:691]: This method is missing Javadoc

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature (hunk 1 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature]: dome

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java (hunk 2 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:10]: done

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java (hunk 3 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java:11]: done

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java (hunk 4 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java]: done

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java (hunk 5 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java:701]: done

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java
[the remark this answers is not in the document]
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/ApplicantProfileStepsImpl.java:1]: done

**Review by katm007 — APPROVED** [review]
reviewed and approved

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java (hunk 2 above)
Comment (hamidnesar) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:10]: @lfcabrera14 this empty line is still present. line 10

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java (hunk 3 above)
Comment (hamidnesar) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/OWM_Vacancy_Manager_Steps.java:11]: empty line 11

**Review by hamidnesar — CHANGES_REQUESTED** [review]
There are still two empty lines

**Review by hamidnesar — APPROVED** [review]
looks good

**Review by sarwarahmed1 — APPROVED** [review]

**Pull request merged**